Plugins/JExtractSwiftPlugin/JExtractSwiftPlugin.swift

```diff
@@ -114,6 +114,13 @@ struct JExtractSwiftBuildToolPlugin: SwiftJavaPluginProtocol, BuildToolPlugin {
       outputSwiftDirectory.appending(path: "\(sourceModule.name)Module+SwiftJava.swift")
     ]
 
+    // Append any JNI cache files
+    if configuration?.mode == .jni {
+      outputSwiftFiles += [
+        outputSwiftDirectory.appending(path: "\(sourceModule.name)+JNICaches.swift")
```

Comment:
Do we really need a new file for this you think?

Reply:
No not really, since it is codegen. I can put it in the library file instead?